Yoast/wordpress-seo
# Disabled marker buttons for divi page builder

## Summary

This PR can be summarized in the following changelog entry:

* Marker buttons are now disabled when the divi builder is active.

## Relevant technical choices:

* Added specific logic for divi, because every page builder handles the classic editor differently.

## Test instructions

This PR can be tested by following these steps:

* Install divi theme and builder (ask me (@xyfi) or @CarolineGeven for the zip files).
* Create a post that has a marker (e.g. keyword density(
* Switch between divi builder, classic editor (editor and text-view), and make sure the marker buttons are disabled when the divi builder and the text view are used, but not when the classic editor is open.

## Quality assurance


Fixes #10361
